mfr/core.py

```diff
@@ -338,7 +338,10 @@ def copy_dir(src, dest):
     except shutil.Error as err:
         logger.warn(err)
     except OSError as err:
-        logger.debug('Skipping {src} (already exists)'.format(src=src))
+        if os.path.exists(src):
```

Comment:
This will always be `True`. The `src` file will always exist. You should check the existence of `dest`.